src/test/java/teammates/test/cases/logic/AccountsLogicTest.java

```diff
@@ -456,4 +456,11 @@ public void testDeleteAccountCascade_nonExistentAccount_shouldPass() {
         // other irrelevant instructors remain
         assertNotNull(logic.getInstructorForEmail(instructor1OfCourse1.getCourseId(), instructor1OfCourse1.getEmail()));
     }
+
+    @Test
+    public void testGetCourseInstitute() {
+        assertNull(accountsLogic.getCourseInstitute("0"))
+        assertEquals(accountsLogic.getCourseInstitute("1"), "dev")
+        assertEquals(accountsLogic.getCourseInstitute("2"), "TEAMMATES Test Institute 1")
+    }
```

Comment:
I suggest to split these "asserts" to another test methods.
It'd be better to read it in the future.
Now all cases are in one method.